src/main/c/netty_quic_boringssl.c

```diff
@@ -787,7 +787,7 @@ jint netty_boringssl_JNI_OnLoad(JNIEnv* env, const char* packagePrefix) {
 
 
     NETTY_JNI_UTIL_LOAD_CLASS(env, byteArrayClass, "[B", done);
-    NETTY_JNI_UTIL_LOAD_CLASS(env, stringClass, "Ljava/lang/String;", done);
+    NETTY_JNI_UTIL_LOAD_CLASS(env, stringClass, "java/lang/String", done);
```

Comment:
why this change ?

Reply:
I had to change this because otherwise it was throwing an exception and crashed, there is more information on this page:
https://developer.android.com/training/articles/perf-jni#faq:-why-didnt-findclass-find-my-class

It seems that on the desktop it still works without `L` and `;`, The unit tests are all successful.

Reply:
ok fair enough...  Thanks a lot!